data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -713,13 +714,13 @@ Scroll (Charm Person)												SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Charm P
 Scroll (Color Spray)												SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Color Spray	TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SPROP:15' Cone stuns creatures based on HD (5+ stunned for 1 round, 4- also blind & stunned for 1d4 rounds, 2- also unconscious for 2d4 rounds. Will DC 11 negates(PH 210)	SOURCEPAGE:p.237
 Scroll (Color Spray/Wizard/2nd)		SPELLS:Magic Item|TIMES=1|CASTERLEVEL=2|Color Spray	TYPE:Magic.Scroll.Arcane.Consumable	COST:50	WT:0.01	SPROP:15' Cone stuns creatures based on HD (5+ stunned for 1 round, 4- also blind & stunned for 1d4 rounds, 2- also unconscious for 2d4 rounds. Will DC 11 negates(PH 210)	SOURCEPAGE:p.237
-Scroll (Comprehend Languages/Arcane)	SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Comprehend Languages	OUTPUTNAME:Scroll (Comprehend Languages)		TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SOURCEPAGE:p.237
+Scroll (Comprehend Languages/Arcane)	SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Comprehend Languages	OUTPUTNAME:Scroll (Comprehend Languages)		TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SOURCEPAGE:p.237	SPROP:Understand spoken or written words for 10 minutes
 Scroll (Confusion/Lesser)			SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Confusion (Lesser)	OUTPUTNAME:Scroll (Confusion, Lesser)		TYPE:Magic.Scroll.Arcane.Consumable	COST:50	WT:0.01	SOURCEPAGE:p.237
 Scroll (Cure Light Wounds/Arcane)		SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Cure Light Wounds	OUTPUTNAME:Scroll (Cure Light Wounds)		TYPE:Magic.Scroll.Arcane.Consumable	COST:50	WT:0.01	SOURCEPAGE:p.237	SPROP:Cures 1d8+1 damage (PH P.215)
 Scroll (Detect Secret Doors)											SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Detect Secret Doors	TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SOURCEPAGE:p.237
 Scroll (Detect Undead/Arcane)			SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Detect Undead	OUTPUTNAME:Scroll (Detect Undead)			TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SOURCEPAGE:p.237
-Scroll (Disguise Self/Lvl 1)			SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Disguise Self	OUTPUTNAME:Scroll (Disguise Self)			TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SPROP:You make yourself and your equipment look different - 1 foot shorter/taller, thin/fat, add/obscure minor feature or look like an entirely different person. +1 disguise for %1 minutes|CASTERLEVEL*10	SOURCEPAGE:p.237
-Scroll (Endure Elements/Arcane)		SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Endure Elements	OUTPUTNAME:Scroll (Endure Elements)			TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SOURCEPAGE:p.237
+Scroll (Disguise Self/Lvl 1)			SPELLS:Magic Item|TIMES=1|CASTERLEVEL=1|Disguise Self	OUTPUTNAME:Scroll (Disguise Self)			TYPE:Magic.Scroll.Arcane.Consumable	COST:25	WT:0.01	SPROP:You make yourself and your equipment look different - 1 foot shorter/taller, thin/fat, add/obscure minor feature or look like an entirely different person. +1 disguise for 10 minutes	SOURCEPAGE:p.237
```

Comment:
This makes sense since the scroll is 1st level.

Reply:
? is that confirmation that it's ok, or an indication that it needs changing somehow?

Reply:
Giving a thumbs up. This change was a smart one and I approve.